### ISSUE TITLE
Better description of how to troubleshoot and reporting errors due diligence

### ISSUE THREAD

**Issue by oesteban** [poldracklab/fmriprep#1849]
With the increase in users, the feedback we receive is also more unclear. Most of our support interactions start with some general guidelines to find out the exact cause of problems and paths where the user can find further information.

Perhaps we could create this documentation based on GitHub's issue template.

**Comment by emdupre** [comment]
Sorry, just to make sure I understand -- is this proposing to create a GitHub issue template to help make sure that users are providing necessary information in e.g. bug reports or feature requests ?

If so, I am a :100: fan of this ! My additional suggestion would be to consider setting an issue template in NeuroStars, since we receive so many questions there. We could do this through [Discourse topic templates](https://meta.discourse.org/t/what-are-topic-templates/38295).

**Comment by oesteban** [comment]
Yup, updating the issue template would fall under this request. But I was more thinking about a particular section on the documentation where we give extended descriptions of what to do when diagnosing problems and then link the issue template.

I'll check on adding a template to NeuroStars too.

**Comment by emdupre** [comment]
Just to confirm, it seems that we don't have an issue template at all right now ?
